src/Raven.Studio/typescript/models/database/tasks/sql/sqlMigration.ts

```diff
@@ -145,8 +165,14 @@ class sqlMigration {
             const primaryKeyColumns = columns.filter(c => _.includes(tableDto.PrimaryKeyColumns, c.sqlName));
             const specialColumnNames = this.findSpecialColumnNames(dbSchema, tableDto.Schema, tableDto.TableName);
             const primaryKeyColumnNames = primaryKeyColumns.map(x => x.sqlName);
-            
-            table.documentColumns(columns.filter(c => !_.includes(specialColumnNames, c.sqlName) && !_.includes(primaryKeyColumnNames, c.sqlName)));
+            if (this.advanced.includeUnsupported() === false) {
```

Comment:
please reverse condition to avoid === false

Reply:
done